card/mod/03_machines/lib/stylesheets/style_cards.scss

```diff
@@ -49,28 +49,92 @@ Choosing (b) will mean your CSS will not be affected by automated updates.
 /* -- card-headers and menus--*/
 
 .card-header {
+  position: relative;
   .card-header-title {
     display: inline-block;
     overflow: hidden;
+    span.glyphicon {
```

Comment:
since these are the non-bootstrap rules, perhaps this should be a non-bootstrap selector?